README.md

```diff
@@ -60,7 +67,11 @@ rm -rf /var/lib/apt/lists/*
 apt-get update
 ```
 
-##### OpenSUSE LEAP & Tumbleweed install
+#### RPM based Package Repository (GNU/Linux x86/x64)
+
+> Detailed or alternative steps in the [instructions by Cloudsmith](https://cloudsmith.io/~balena/repos/etcher/setup/#formats-rpm)
```

Comment:
Should this have been added to the "Redhat (RHEL) and Fedora-based Package Repository (GNU/Linux x86/x64)" section?

Reply:
zypper (opensuse) works with rpm too. From this line all `sh snippets` have the script with rpm.
At the same time, yes, it seems logical to have Redhat first in the rpm section.